### PR TITLE
Optimizing the AgentCore Integration Test Script

### LINKED ISSUE #1578: Optimize the action script of the integration test
### What would you like to be enhanced?

1、Modify the integration test script and adjust the script execution conditions.

### Why is this needed?

1、No function modification is involved, and the script is still executed. As a result, the overall execution time of the integration test script is too long.

### PR DESCRIPTION
**Pull request by hanbingleixue**
**What type of PR is this?**

Integration Test.

**What this PR does / why we need it?**

1、A  script is provided for the integration test of services in the AgentCore.
2、The agentCore is not added when the integration script execution conditions of some plug-ins are modified.

**Which issue(s) this PR fixes？**

Fixes #1578

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/sermant-io/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [x] GitHub Actions works fine in this PR.

### CODE REVIEW

File: .github/workflows/agentcore_service_test.yml [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+          key: ${{ runner.os }}-nacos-server-2.1.0
+          restore-keys: |
+            ${{ runner.os }}-nacos-server-2.1.0
+      - name: download nacos
+        run: |
+          export ROOT_PATH=$(pwd)
+          bash ./sermant-integration-tests/scripts/tryDownloadMidware.sh nacos210
+  build-agent-and-cache:
+    name: build agent and cache
+    runs-on: ubuntu-latest
```
Comment (lilai23) [inline .github/workflows/agentcore_service_test.yml:87]: add a condition judgment about whether this job should be executed

Comment (hanbingleixue) [reply .github/workflows/agentcore_service_test.yml:87]: done

File: .github/actions/scenarios/xds-service/xds-service-discovery/client-envoy/action.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@ runs:
       shell: bash
       run: |
         kubectl delete -f sermant-integration-tests/xds-service-test/script/discovery/spring-server-sermant-xds-open.yaml
-        sleep 3s
+        sleep 8s
```
Comment (daizhenyu) [inline .github/actions/scenarios/xds-service/xds-service-discovery/client-envoy/action.yml]: Delay can be unified

**Review by lilai23 — APPROVED** [review]

**Review by daizhenyu — APPROVED** [review]

File: .github/actions/scenarios/xds-service/xds-service-discovery/client-envoy/action.yml (hunk 2 above)
Comment (hanbingleixue) [reply .github/actions/scenarios/xds-service/xds-service-discovery/client-envoy/action.yml]: done

**Pull request merged**